### PR TITLE
Okapi Dependency Update from V. 0.36 to 1.43.0 

### PR DESCRIPTION
**Pull request by hylstonnb**
Updating Okapi dependency to the latest version which supports java 8. (1.43.0)
Currently the latest version of Okapi is 1.45.0

### CODE REVIEW

**Comment by aurambaj** [comment]
Thank you @hylstonn! this is promising. Would you mind updating the commit message with more detail about each change required?

File: common/src/main/java/com/box/l10n/mojito/okapi/filters/JSONFilter.java [hunk 1]
```diff
@@ -10,8 +10,8 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import net.sf.okapi.common.Event;
-import net.sf.okapi.common.annotation.XLIFFNote;
-import net.sf.okapi.common.annotation.XLIFFNoteAnnotation;
+import net.sf.okapi.common.annotation.Note;
+import net.sf.okapi.common.annotation.NoteAnnotation;
```
[the remark this answers is not in the document]
Comment (hylstonnb) [reply common/src/main/java/com/box/l10n/mojito/okapi/filters/JSONFilter.java:14]: its a replacement, XLIFFNoteAnnotation was deprecated and got removed already.

File: cli/src/test/resources/com/box/l10n/mojito/cli/command/ImportLocalizedAssetCommandTest_IO/importYamlWithExtractFields/expected/demo_fr-CA.yaml [hunk 2]
```diff
@@ -64,4 +64,4 @@ components:
       schema:
         type: string
         pattern: ^\d+$
-        maxLength: 18
\ No newline at end of file
+        maxLength: 18
```
[the remark this answers is not in the document]
Comment (hylstonnb) [reply cli/src/test/resources/com/box/l10n/mojito/cli/command/ImportLocalizedAssetCommandTest_IO/importYamlWithExtractFields/expected/demo_fr-CA.yaml:67]: yes, the source file contains the extra line. Probably a bug they fixed in Okapi

**Review by aurambaj — APPROVED** [review]
Looks good, thank you!

**Pull request merged**